src/fileinfo/file_presentation/json_presentation.cpp

```diff
@@ -299,6 +299,26 @@ void JsonPresentation::presentPatterns(Json::Value &root) const
 	root["patterns"] = patterns;
 }
 
+/**
+ * Present information about missing dependencies
+ * @param root Parent node in output document
+ */
+void JsonPresentation::presentMissingDepsInfo(Json::Value &root) const
+{
+	if (returnCode == ReturnCode::FILE_NOT_EXIST || returnCode == ReturnCode::UNKNOWN_FORMAT)
+	{
+		return;
+	}
+
+	const auto numberOfMissingDeps = fileinfo.getNumberOfMissingDeps();
+
+	Value jMissingDeps;
+	jMissingDeps["count"] = (unsigned int)numberOfMissingDeps;
```

Comment:
One thing with `fileinfo` is that we present everything in JSON as strings and I think everyone working with fileinfo JSONs kinda expects this. Would it be possible to present this value as string too?